### PR TITLE
Create Set-SPOTenantRenameSitePrioritization.md

### PR DESCRIPTION
**Pull request by AnfraMsft**
Documentation for new cmdlet to prioritize individual sites within Advanced Tenant Rename

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ms.reviewer: anushmag
+manager: anushmag
+schema: 2.0.0
+ms.topic: reference
+---
+
+# Set-SPOTenantRenameSitePrioritization
+
+## SYNOPSIS
+This cmdlet allows prioritization of a site for early execution, as part of [Advanced Tenant Rename](https://aka.ms/advancedtenantrename).
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:18]: >This cmdlet allows [](http://example.com/codeflow?start=0&length=18)

```suggestion
Allows
```

Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:18]: >https://aka.ms/advancedtenantrename [](http://example.com/codeflow?start=101&length=35)

replace this aka.ms with the relative URL `/sharepoint/change-your-sharepoint-domain-name#advanced-tenant-rename-preview`

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## SYNTAX
+
+```
+Set-SPOTenantRenameSitePrioritization -SiteUrl <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+As part of [Advanced Tenant Rename](https://aka.ms/advancedtenantrename), organizations can prioritize up to 4,000 sites in their scheduled rename.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:27]: First add a statement that explains what the cmdlet does. Further descriptions will come after that statement.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Prioritizing a site means that it will be queued for initial execution among all other sites in your organization as part of the overall rename operation.
+
+To run this cmdlet, you need SharePoint admin permissions. Also, it is possible to start prioritizing sites only once the rename operation has been scheduled using the [Start-SPOTenantRename](https://learn.microsoft.com/powershell/module/sharepoint-online/start-spotenantrename?view=sharepoint-ps) cmdlet.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:31]: >https://learn.microsoft.com/powershell/module/sharepoint-online/start-spotenantrename?view=sharepoint-ps [](http://example.com/codeflow?start=192&length=104)

`/start-spotenantrename?view=sharepoint-ps`

Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:31]: >To run this cmdlet, you need SharePoint admin permissions.  [](http://example.com/codeflow?start=0&length=59)

remove this, then add this line at the end of this block:

`You must be a SharePoint or Global Administrator to run this cmdlet.`

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Please note that prioritizing a site is not a guarantee that it will complete first. There are several factors that can affect processing times, and multiple site renames are processed in parallel. Prioritized sites have a much higher chance of completing first.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-SPOTenantRenameSitePrioritization -SiteUrl https://contoso.sharepoint.com/sites/projectx
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:39]: >PS C:\> Set-SPOTenantRenameSitePrioritization [](http://example.com/codeflow?start=0&length=45)

```suggestion
Set-SPOTenantRenameSitePrioritization
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+This example prioritizes the 'projectx' site within the Advanced Tenant Rename.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:42]: > 	 [](http://example.com/codeflow?start=79&length=2)

remove trailing space and tab

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md (hunk 1 above)
Comment (samkabue) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md:18]: applies to this  place and others

**Comment by AnfraMsft** [comment]
@samkabue I believe I've incorporated the suggestions. Could you please review?

**Review by samkabue — APPROVED** [review]
:shipit:

**Pull request merged**